wdl/SplitVcf.wdl

```diff
@@ -0,0 +1,145 @@
+version 1.0
+    
+import "Structs.wdl"
+import "TasksMakeCohortVcf.wdl" as MiniTasks
+
+
+workflow GetBatchedVcf {
```

Comment:
This workflow isn't very efficient - there's no need to scatter the vcf. The vcf traversal (compressing/parsing/decompressing) is usually the bottleneck in terms of speed - the subset operation itself is extremely fast. So you're traversing the vcf twice here but just subsetting on the first pass would take about the same amount of compute. This whole workflow can be replaced with `SubsetVcfBySamplesList` with `keep_af=True` and `remove_private_sites=False`.

Reply:
Good point! I changed it accordingly.